how-to-guides/data-versioning/notebooks/Organize_and_share_dataset_versions.ipynb

```diff
@@ -107,7 +110,7 @@
     "1. Get your [Neptune API token](https://docs.neptune.ai/getting-started/installation#authentication-neptune-api-token)\n",
     "2. Pass the token to `api_token` argument of the `neptune.init_project()` method: `api_token=YOUR_API_TOKEN`\n",
     "3. Get your [Neptune project name](https://docs.neptune.ai/getting-started/installation#setting-the-project-name)\n",
-    "4. Pass your project to the `name` argument of the `neptune.init_project()` method.\n",
+    "4. Pass your project to the `project` argument of the `neptune.init_project()` method.\n",
```

Comment:
Noticed a mistake below on line 118: it has `init_run` instead of `init_project`. Might be good to fix while we're at it.

Reply:
good catch! thanks!